App/ui.r

```diff
@@ -159,10 +159,17 @@ tagList(
             "MixFishConsiderations",
             tabsetPanel(
                 tabPanel(
-                    "Fishing effort",
+                    sidebarLayout(
+                      "Fishing effort",
+                      sidebarPanel = sidebarPanel(shinyjqui::orderInput("order", 
+                                                                        label = "Drag boxes to determine fleet order",
+                                                                        items = c("Country", "Vessel size", "Gear"), 
+                                                                        item_class = "primary" 
+                      )),
+                      mainPanel = mainPanel(column(width = 9,
                     tags$style(type = "text/css", "#fleet_stock_effort_plot {height: calc(99vh - 220px) !important;} overflow-y: hidden;"),
                     plotlyOutput("fleet_stock_effort_plot", height = "100%", width = "50%")
-                ),
+                ))),
```

Comment:
It's not running for me, I think there is a problem of parentheses

Warning: Error in sourceUTF8: Error sourcing C:\Users\LUCA~1.LAM\AppData\Local\Temp\RtmpS4tmLr\file5298366b74c4
  [No stack trace available]
Error in parse(file, keep.source = FALSE, srcfile = src, encoding = enc) :
  D:\Profile\Documents\GitHub\mixfish_fo_app_dev\App/ui.R:276:0: unexpected end of input
274:     )
275: )
    ^

Reply:
It should be fixed now